commit f85405d733dd4f1cf05879eab3b852ce81eab3ea (refs/changes/98/4798/5)
Author: dyep <dyep49@gmail.com>
Date:   2018-10-18T15:08:22-07:00 (12 months ago)
    
    OS-7220 Snapshots are not unmounted on VM pre-halt
    Reviewed by: Rob Johnston <rob.johnston@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/overlay/generic/usr/lib/brand/jcommon/statechange b/overlay/generic/usr/lib/brand/jcommon/statechange
index da3bd811..58eae5ab 100644
--- a/overlay/generic/usr/lib/brand/jcommon/statechange
+++ b/overlay/generic/usr/lib/brand/jcommon/statechange
@@ -876,7 +876,7 @@ pre)
 		setup_fs
 		;;
 	4)	# pre-halt
-		[[ -n "$jst_snowsnap" ]] && cleanup_snapshots
+		[[ -n "$jst_showsnap" ]] && cleanup_snapshots
 		cleanup_net
 		;;
 	esac
